.github/PR_TEMPLATE/pull_request.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Pull Request
+about: Increment new features to kool.dev
+title: ""
+labels: ""
+assignees: ""
+---
+
+**This PR handles/relates to issue #number-of-the-issue-here (if you have no issue, please consider opening one for history tracking).**
+
+-   [ ] :beetle: Bug Fix
+-   [ ] :trophy: Feature
+-   [ ] :pencil: Refractor
+-   [ ] :warning: Break Change
+-   [ ] :new: Other
+
+#### TL;DR
+
+Provide a short description on what you are implementing. Reference errors, or anything that makes it very clear after a single read what is being changed. Focus on write less, while making yourself clear.
+
+<hr>
+
+#### Describe the update
+
+A clear explanation of what is being updated. Try to remember/respond to these points:
+
+:white_check_mark: I will make sure my PR **passes** all CI checks **(and is tested)**.
+
+:cool: If I found an issue, I have already open a issue or alerted the maintainers, heterofore submitting this PR for review.
+
+-   _Why is this Important?_
+-   :warning: **Is this introducing a Breaking Change?**
+-   _Are there side effects ? (if so list them)_
+-   _Is there something relevant to a specific commit ?_
+-   _Are you solving a issue? if so don't forget to note which and explain your solution process_
+
+#### :dart: Kool Target version and environment
```

Comment:
This section is not quite there yet. Instead of this:

![image](https://user-images.githubusercontent.com/1116377/101793680-d37bea00-3ae4-11eb-8c1d-0f9eb9d61e06.png)

We really simply need:

`Target version: [ ] next patch release [ ] next minor release [ ] next major release`